rust/generate_message/src/parser.rs

```diff
@@ -471,14 +471,14 @@ impl std::fmt::Display for Set {
 #[derive(clap::Args, Debug)]
 pub struct Make {
     /// payload
-    #[clap(long, name = "msg", value_parser)]
```

Comment:
`value_parser` is a default value => no need to specify. 
`Name` is the same as a variable name => no need to specify.